contracts/attribute/OwnerDelegatable.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.13;
+
+import {IVotes} from "@openzeppelin/contracts/governance/utils/IVotes.sol";
```

Comment:
nit: Our code style, even in this library which hasn't been updated for explicit imports, pads spaces between the imported type and the curly braces; see `IKept.sol` or `IVerifierBase.sol` for example.